renovate.json5

```diff
@@ -1,6 +1,9 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "github>blumilksoftware/traefik-configuration/renovate/presets/default.json5",
+    "github>blumilksoftware/traefik-configuration//renovate/presets/default.json5",
```

Comment:
`blumilksoftware/traefik-configuration` is a private repository. Maybe we should create a separate one for things like this?

Reply:
Renamed to `infrastructure` as suggested.